app/design/adminhtml/default/default/template/dashboard/graph.phtml

```diff
@@ -12,34 +12,17 @@
  * @copyright   Copyright (c) 2021-2022 The OpenMage Contributors (https://www.openmage.org)
  * @license     https://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
  */
-/*
+
+/**
+ * @var Mage_Adminhtml_Block_Dashboard_Graph $this
+ */
+
 ?>
-<div style="width:587px;margin:20px;position:relative;margin-left:-294px;left:50%">
-    <div style="position:absolute;right:1px;top:0px;left: expression(parentNode.offsetWidth - offsetWidth - 1 + 'px');">
-        <div style="padding:5px;height:15px;width:15px;background-color:#F4F4F4" id="dashboard_<?php echo $this->getHtmlId() ?>_cal_div"><img src="<?php echo Mage::getDesign()->getSkinUrl('images/grid-cal.gif') ?>" onclick="toggleCal('<?php echo $this->getHtmlId() ?>')" /></div>
-        <div style="padding:5px;display:none;width:200px;" id="dashboard_<?php echo $this->getHtmlId() ?>_range_div" class="switcher">
-            <?php echo $this->__('Select Range') ?>:
-            <select name="period" id="order_<?php echo $this->getHtmlId() ?>_period" onchange="changePeriod(this)">
-                <?php foreach ($this->helper('adminhtml/dashboard_data')->getDatePeriods() as $_value=>$_label): ?>
-                    <?php if(in_array($_value, array('custom'))) continue; ?>
-                    <option value="<?php echo $_value ?>" <?php if($this->getRequest()->getParam('period')==$_value): ?> selected<?php endif ?>><?php echo $_label ?></option>
-                <?php endforeach ?>
-            </select>
-            <img src="<?php echo Mage::getDesign()->getSkinUrl('images/dashboard-close.gif') ?>" onclick="toggleCal('<?php echo $this->getHtmlId() ?>')" />
-        </div>
-    </div>
-    <?php if($this->getCount()): ?>
-    <img src="<?php echo $this->getChartUrl() ?>" />
-    <?php else: ?>
-    <div align="center" style="width:587px;height:300px;"><?php echo $this->__('No Data Found') ?></div>
-    <?php endif ?>
-</div>
-<?php */ ?>
```

Comment:
Is it OK to remove this part of the code?

Reply:
It is not used.

Reply:
Please mark as resolved to move on.